theme/mathform.css

```diff
@@ -1,3 +1,7 @@
+.ck.ck-rounded-corners > .ck-balloon-panel {
+	border-radius: 4px;
```

Comment:
maybe we should override `--ck-border-radius` for entire CKEditor?

Reply:
i ovverride `--ck-border-radius`, a lot has not changed in the editor, but balloons look better and consistent